adapters/ix/ix.go

```diff
@@ -100,6 +127,129 @@ func (a *IxAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters
 	return append(requests, multiSizeRequests...), errs
 }
 
+func handleMultiImpInSingleRequest(request *openrtb2.BidRequest, a *IxAdapter) ([]*adapters.RequestData, []error) {
+	nImp := len(request.Imp)
+	if nImp > a.maxRequests {
+		request.Imp = request.Imp[:a.maxRequests]
+		nImp = a.maxRequests
+	}
+	// Multi-size banner imps are split into single-size requests.
+	// The first size imp requests are added to the first slice.
+	// Additional size requests are added to the second slice and are merged with the first at the end.
+	requests := make([]*adapters.RequestData, 0, a.maxRequests)
+	multiSizeRequests := make([]*adapters.RequestData, 0, a.maxRequests-1)
+	errs := make([]error, 0)
+
+	if err := BuildIxDiag(request); err != nil {
+		errs = append(errs, err)
+	}
+
+	headers := http.Header{
+		"Content-Type": {"application/json;charset=utf-8"},
+		"Accept":       {"application/json"}}
+
+	siteIds := make(map[string]bool)
+	originalImps := make([]openrtb2.Imp, 0, len(request.Imp))
+	additionalMultiSizeImps := make([]openrtb2.Imp, 0)
+	requestCopy := *request
+	for _, imp := range requestCopy.Imp {
+		if err := parseSiteId(&imp, siteIds); err != nil {
+			errs = append(errs, err)
+			continue
+		}
+
+		if err := moveSid(&imp); err != nil {
+			errs = append(errs, err)
```

Comment:
To Confirm: It is your intent to continue with the auction if `moveSid` fails? Not sure if omitting the `continue` statement here is intentional or an oversight.

Reply:
Confirming it is intentional to still make the bidrequest if that function errors out